ddl/cancel_test.go

```diff
@@ -232,6 +232,8 @@ func TestCancel(t *testing.T) {
 
 	// Prepare schema.
 	tk.MustExec("use test")
+	// ToDo: will check why tidb_ddl_enable_fast_reorg could not be default on in another pr.
```

Comment:
```suggestion
	// TODO: Will check why tidb_ddl_enable_fast_reorg could not be defaulted on in another PR.
```

Please update other same comments.